common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -148,6 +148,12 @@ div.problem {
       margin-top: $baseline;
     }
   }
+
+  span > label {
+    display: block;
+    margin-bottom: $baseline;
+    font: inherit;
+  }
```

Comment:
@chris-mike Can you please give it a quick look.
This label was a `p` tag earlier and it contains question title/text, the styling was disturbed when we changed it to `lable`.